src/govuk/components/footer/_footer.scss

```diff
@@ -196,9 +196,9 @@
     }
   }
 
-  // Sections two-third:one-third on desktop
+  // If the there are only two sections and set the layout to be two-third:one-third on desktop
   @include mq ($from: desktop) {
-    .govuk-footer__section:first-child {
+    .govuk-footer__section:first-child:nth-last-child(2) {
```

Comment:
Won't this apply a `flex-grow: 2` to the 2nd-to-last child of the footer section _regardless_ of how many columns there are?

I.e. a three-column layout will set the middle column to `flex-grow: 2` - is that expected?

Reply:
nth-last-child counts backwards, so it'll only target the first element and the 2nd from last element.

if you had three columns then the 2nd element would not also be the first element

Reply:
Ahhhhhh that makes sense. Clever! 👏

Reply:
I'm not a fan of clever code but feels appropriate here

Reply:
I'll try and write a comment that explains this better...

Reply:
Thanks, this is much better!